### DISCUSSION TITLE
How can I insert new candidates to already open minibuffer?

### DISCUSSION THREAD

**Discussion by wybryk-ananasa** [minad/vertico#434] [Q&A]
I would like to insert new candidates into already open buffer.
How can I do it?
<img width="397" alt="Screenshot 2024-01-18 at 20 07 39" src="https://github.com/minad/vertico/assets/144451734/01b992dc-78e8-4735-90a8-f65afe713d7d">

**Comment by minad** [comment]
This question is out of scope/too generic for the Vertico discussion board. I suggest to read the documentation of the completion mechanism, see https://www.gnu.org/software/emacs/manual/html_node/elisp/Completion.html. You can create your own custom completion table (which stores the candidates as a list for example) and implement separate commands which add to this list.
